resources/zib-MedicationContraIndication.xml

```diff
@@ -0,0 +1,176 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-MedicationContraIndication" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicationContraIndication" />
+  <name value="ZibMedicationContraIndication" />
+  <title value="zib MedicationContraIndication" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A contra-indication (CI) for medication safety is a condition or characteristic of a patient where healthcare professionals are not allowed to subscribe certain medicines, or only under certain conditions." />
+  <purpose value="This Flag resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) MedicationContraIndication v1.0 (2020)](https://zibs.nl/wiki/MedicationContraIndication-v1.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-medicationcontraindication-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/MedicationContraIndication-v1.0(2020EN)" />
+    <name value="zib MedicationContraIndication-v1.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Flag" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Flag" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Flag">
+      <path value="Flag" />
+      <short value="MedicationContraIndication" />
+      <comment value="A flag is a warning or notification of some kind presented to the user - it could be a physician or any other person involved in patient care. It usually represents something of sufficient significance to warrant a special representation of some kind - rather than just a note in the source. A flag has a subject that represents the resource that will trigger its display.&#xD;&#xA;&#xD;&#xA;Status is not defined in the zib and it depends on the concepts ReasonClosure and EndDate. When the concepts ReasonClosure and EndDate are filled in, status is given a value that matches the situation." />
```

Comment:
Status is not defined in the zib and it depends on the concepts ReasonClosure and EndDate. When the concepts ReasonClosure and EndDate are filled in, status is given a value that matches the situation.

Misschien zoiets?

The zib MedicationContraIndication defines no status. However, the concepts ReasonClosure and EndDate relate to the Flag inactive status. Therefore it is expected that Flag status gets a matching value if ReasonClosure is present or the EndDate is in the past.